requirements/prod.txt

```diff
@@ -11,19 +11,20 @@ MarkupSafe==0.23 \
 django-csp==3.0 \
     --hash=sha256:e6e627955651235852508f98238f3f04c7ab9a77b625a6bd101cd0cf1a1e3419 \
     --hash=sha256:e44479be426ffd5371b7522f43f8c32f8597a5003c236f31bb98ff4497a20a21
-pytz==2015.4 \
-    --hash=sha256:9966d2c3f86b23374da9890584f16b03f6e46055824e30a108716974d45d1e45 \
-    --hash=sha256:f7c5be4db86f1a76ccc4c807f9008536275abec3e2815d18265d694aeef3cb3c \
-    --hash=sha256:6ce918e48daed649e13aa4eb2ce99d80212411401be42288c94473a81109244e \
-    --hash=sha256:6441e798f8863744c90574de253bed328c477fd2c86caf52b8d9d85f1ecb9f96 \
-    --hash=sha256:4d64ed1b9e0e73095f5cfa87f0e97ddb4c840049e8efeb7e63b46118ba1d623a \
-    --hash=sha256:39ef0541e489b1cf5515cedf1b1f99de585fd660463cdcde43a03d7d92042782 \
-    --hash=sha256:685b1994026572177458af143cc24a2082f400f18e63fdc8e2ed68dd93bb0f49 \
-    --hash=sha256:8446013dfbde9b3f4f9bfea09418ddd526760a8798c41badd3f707a98ad0745b \
-    --hash=sha256:d5ff706089e9150760d238b66203a24d4a0a6175160410191995ff04d1893faa \
-    --hash=sha256:a78b484d5472dd8c688f8b3eee18646a25c66ce45b2c26652850f6af9ce52b17 \
-    --hash=sha256:c4ee70cb407f9284517ac368f121cf0796a7134b961e53d9daf1aaae8f44fb90 \
-    --hash=sha256:00f35fb0e7e2fc348cd091d25d36c6768b721463d12396d68a9491a8a5f91ecf
+pytz==2016.10 \
```

Comment:
Same here. Celery 4 needs most of these dependencies upgraded, but once we rebase with 3.1.25 these will longer be part of this commit.